at-quannguyen/W4/ex3.rb

```diff
@@ -0,0 +1,32 @@
+class ArrayClone
+  attr_accessor :array, :length
+  def initialize
+    self.array = []
+    self.length = 0
+  end
+
+  def map(&block)
+    result = []
+    @array.each { |e| result << block.(e) }
+    result
+  end
+
+  def select(&block)
+    result = []
+    @array.each { |e| result << i if block.(e) }
+    result
+  end
+
+  def length
+    @array.each { |e| @length += 1 }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/cd24ce24c04c57d4b75cc24b8a37d518f57e5e0e/at-quannguyen/W4/ex3.rb#L21"></span>Lint/UnusedBlockArgument: Unused block argument - <code>e</code>. You can omit the argument if you don’t care about it. (https://github.com/rubocop-hq/ruby-style-guide#underscore-unused-vars)</td>
    </tr>
  </tbody>
</table>